mysql_exporter/hooks.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import os
+
+from jinja2 import Environment, FileSystemLoader, Template
+from snaphelpers import Snap
+
+from mysql_exporter.log import setup_logging
+
+DEFAULT_CONFIG = {
+    # MySQL Connection information
+    "mysql.user": "exporter",
+    "mysql.password": "exporterpasswd",
```

Comment:
i am not sure i agree with hardcoding these values. also, i am not sure that the user would exist when the hook is called